### PR TITLE
[Merge on 5/23] New AI samples

### PR DESCRIPTION
**Pull request by savannahostrowski**

### CODE REVIEW

**Comment by savannahostrowski** [comment]
cc: @achandmsft @pamelafox for review

File: website/src/data/users.tsx [hunk 1]
```diff
@@ -181,7 +181,34 @@ const Users: User[] = [
...
+    description: 'A sample app for the Retrieval-Augmented Generation pattern running in Azure, using Azure Cognitive Search for retrieval and Azure OpenAI large language models to power ChatGPT-style and Q&A experiences.',
+    preview: require('./images/chatgpt-enterprise-dotnet.png'),
+    website:'https://github.com/Azure-Samples',
+    author:'Azure Content Team',
+    source: 'https://github.com/Azure-Samples/azure-search-openai-demo-csharp/',
+    tags: ['featured', 'bicep','dotnetCsharp','aks', 'kubernetes','aca','cosmosdb','cognitivesearch', 'openai', 'chatgpt', 'rediscache', 'ai'],
+  },
+  {
+    title: 'BYO-Chat Application using Azure OpenAI',
+    description: 'This repository shows you the basic code of how to use the OpenAI SDK for Python to build your own chat application with Azure OpenAI',
```
Comment (pamelafox) [inline website/src/data/users.tsx]: It'll be an actual chat (see https://chatfrance-zypx32dl-ca.livelyrock-dc5184bd.francecentral.azurecontainerapps.io/ ) so maybe

```suggestion
    description: 'A sample chat app that uses Python and the Flask framework to stream ChatGPT completions as server-sent events to a simple frontend.',
```

Comment (pamelafox) [reply website/src/data/users.tsx]: Title could also be tweaked, since I don't know if its BYO at this point, more like "Simple Chat"

Comment (savannahostrowski) [reply website/src/data/users.tsx]: Sounds good! I think I just copied this from the repo description so we probably want to update it there as well!

File: website/src/data/users.tsx [hunk 2]
```diff
@@ -181,7 +181,34 @@ const Users: User[] = [
...
+    //TODO: must update
+    preview: require('./images/chatgpt-enterprise-dotnet.png'),
+    website:'https://github.com/Azure-Samples',
+    author: 'Azure Content Team',
+    source: 'https://github.com/Azure-Samples/chatgpt-quickstart',
+    tags: ['featured', 'bicep','python','openai', 'chatgpt', 'ai']
```
Comment (pamelafox) [inline website/src/data/users.tsx]:
```suggestion
    tags: ['featured', 'bicep','python','openai', 'chatgpt', 'ai', 'flask', 'aca']
```

File: website/src/data/users.tsx (hunk 1 above)
Comment (pamelafox) [reply website/src/data/users.tsx]: Yep, makes sense, I'll update the README in my pending PR

**Comment by savannahostrowski** [comment]
@hemarina Can you click the big green button and merge this in on 5/23 at 9am?

**Pull request merged**